libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -64,10 +64,10 @@ const AP_Param::GroupInfo SRV_Channel::var_info[] = {
+    // @Values{Copter}: -1:GPIO,0:Disabled,1:RCPassThru,6:MountPan,7:MountTilt,8:MountRoll,9:MountOpen,10:CameraTrigger,12:Mount2Pan,13:Mount2Tilt,14:Mount2Roll,15:Mount2Open,22:SprayerPump,23:SprayerSpinner,27:Parachute,28:Gripper,29:LandingGear,30:EngineRunEnable,31:HeliRSC,32:HeliTailRSC,33:Motor1,34:Motor2,35:Motor3,36:Motor4,37:Motor5,38:Motor6,39:Motor7,40:Motor8,51:RCIN1,52:RCIN2,53:RCIN3,54:RCIN4,55:RCIN5,56:RCIN6,57:RCIN7,58:RCIN8,59:RCIN9,60:RCIN10,61:RCIN11,62:RCIN12,63:RCIN13,64:RCIN14,65:RCIN15,66:RCIN16,73:ThrottleLeft,74:ThrottleRight,75:TiltMotorFrontLeft,76:TiltMotorFrontRight,81:BoostThrottle,82:Motor9,83:Motor10,84:Motor11,85:Motor12,88:Winch,90:CameraISO,91:CameraAperture,92:CameraFocus,93:CameraShutterSpeed,94:Script1,95:Script2,96:Script3,97:Script4,98:Script5,99:Script6,100:Script7,101:Script8,102:Script9,103:Script10,104:Script11,105:Script12,106:Script13,107:Script14,108:Script15,109:Script16,120:NeoPixel1,121:NeoPixel2,122:NeoPixel3,123:NeoPixel4,124:RateRoll,125:RatePitch,126:RateThrust,127:RateYaw,129:ProfiLED1,130:ProfiLED2,131:ProfiLED3,132:ProfiLEDClock,133:Winch Clutch,134:SERVOn_MIN,135:SERVOn_TRIM,136:SERVOn_MAX,138:Alarm,139:Alarm Inverted,140:RCIN1_MAPPED,141:RCIN2_MAPPED,142:RCIN3_MAPPED,143:RCIN4_MAPPED,144:RCIN5_MAPPED,145:RCIN6_MAPPED,146:RCIN7_MAPPED,147:RCIN8_MAPPED,148:RCIN9_MAPPED,149:RCIN10_MAPPED,150:RCIN11_MAPPED,151:RCIN12_MAPPED,152:RCIN13_MAPPED,153:RCIN14_MAPPED,154:RCIN15_MAPPED,155:RCIN16_MAPPED,156:RCIN17_MAPPED
```

Comment:
although its precedent for passthru, the mix of camel and all caps is distracting....but it should at least be a bit more explicative.....RCIN1Scaled,etc... MAPPED has no meaning...

Reply:
I agree with @Hwurzburg  about mapped vs scaled. It is actually scaled not mapped. On case - OMG NOT MORE CAPS PLEASE! :-)

Reply:
Fixed and used the RCINxScaled